plugins/org.python.pydev.jython/jysrc/pyedit_marker_example.py

```diff
@@ -0,0 +1,145 @@
+"""This example shows how we can use Eclipse markers in Jython scripts.
+
+PyDev provides following custom set of marker types that can be used
+in scripts without interferences with other marker types:
+
+  `org.python.pydev.jython.problemmarker`
+
+  `org.python.pydev.jython.taskmarker`
+
+  `org.python.pydev.jython.bookmark`
+
+These marker types are derived from standard Eclipse marker types so for more
+details about them please check Eclipse documentation:
+
+https://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Fconcepts%2Fconcepts-11.htm
+"""
+if False:
+    from org.python.pydev.editor import PyEdit #@UnresolvedImport
+    cmd = 'command string'
+    editor = PyEdit
+    systemGlobals = {}
+
+#--------------------------------------------------------------- REQUIRED LOCALS
+#interface: String indicating which command will be executed
+#As this script will be watching the PyEdit (that is the actual editor in Pydev), and this script
+#will be listening to it, this string can indicate any of the methods of org.python.pydev.editor.IPyEditListener
+assert cmd is not None
+
+#interface: PyEdit object: this is the actual editor that we will act upon
+assert editor is not None
+
+# Uncomment the block below then press Ctrl+2 and select add-example-markers
+# or remove-example-markers (accept with <ENTER>).
+# Currently opened file should have at least three lines.
+
+"""
+if cmd == 'onCreateActions':
+    from org.python.pydev.shared_ui.utils import PyMarkerUtils
+
+    action_class = editor.getActionClass()
+
+    class AddMarkersCommand(action_class):
```

Comment:
classes should always be created and then assigned to an entry in systemGlobals

I.e.: 

AddMarkersCommand = systemGlobals.get('AddMarkersCommand')
if AddMarkersCommand is None:
    class AddMarkersCommand(action_class):
        ...
    systemGlobals['AddMarkersCommand'] = AddMarkersCommand

I know this is only an example, but as others may reference it, I think it should use the proper expectation (the problem being that if that's not done, a new class will be created and used whenever a new editor is created, which can be reasonably slow under Jython -- other pyedit_xxx modules should already use this -- you can see pyedit_assign_params_to_attributes.py as an example).

p.s.: Same thing for the RemoveMarkersCommand.